pkg/runtime/security/auth.go

```diff
@@ -109,14 +108,22 @@ func (a *authenticator) CreateSignedWorkloadCert(id, namespace, trustDomain stri
 
 	c := sentryv1pb.NewCAClient(conn)
 
+	token, tokenValidator, err := securityToken.GetSentryToken(true)
+	if err != nil {
+		return nil, fmt.Errorf("error obtaining token: %w", err)
+	}
 	resp, err := c.SignCertificate(context.Background(),
 		&sentryv1pb.SignCertificateRequest{
-			CertificateSigningRequest: certPem,
+			CertificateSigningRequest: csrPem,
 			Id:                        getSentryIdentifier(id),
```

Comment:
This intentionally not just the app ID?  (Instead, being SENTRY_LOCAL_IDENTITY which can be > 64 characters)

Reply:
Hi @a-shoemaker, indeed it has been done to ensure backwards compatibility with a 1.11 sentry server. The intention is to use the app ID in 1.13.